app/services/data_requests/write_cloudwatch_logs.rb

```diff
@@ -43,18 +38,33 @@ def write_row(row)
       multi_factor_auth_method = data.dig(
         'properties', 'event_properties', 'multi_factor_auth_method'
       )
+
+      mfa_key = case multi_factor_auth_method
+      when 'sms', 'voice'
+        'phone_configuration_id'
+      when 'piv_cac'
+        'piv_cac_configuration_id'
+      when 'webauthn'
+        'webauthn_configuration_id'
+      when 'totp'
+        'auth_app_configuration_id'
+      end
+
+      multi_factor_id = data.dig('properties', 'event_properties', mfa_key)
```

Comment:
This will present just the plain ID, but I have the same worry that we have collisions across various integer primary keys. Another option would be to prefix? Ex: `phone_configuration_id:1234`, so this would be something like...


```suggestion
      row_id = data.dig('properties', 'event_properties', mfa_key)
      multi_factor_id = multi_factor_id && "#{mfa_key}:#{row_id}"
```

Reply:
I think the latter is better for a report like this since I'm assuming the person receiving the report won't be using the IDs to do other queries and need to join, etc.

If they need more information on `phone_configuration:1234`, they'd give that to us and we would figure it out from there?

Reply:
Great, I'll update to use the compound key. Thanks!